noxfile.py

```diff
@@ -249,3 +263,16 @@ def format(session):
     session.run("black", *LINT_FILES)
     session.run("blackdoc", *LINT_FILES)
     session.run("isort", *LINT_FILES)
+
+
+@nox.session
+def clean(session):
+    """Remove temporary files"""
+    tracked_files_output = get_example_other_files()
+    for ifile in tracked_files_output:
+        os.remove(ifile)
+    flist = glob.glob("examples/*")
```

Comment:
Is glob only listing folders or also files here?

Reply:
Also files but then you check the len, it should skip files since they have len>0